packages/tools/src/tools/annotation/PlanarFreehandROITool.ts

```diff
@@ -298,10 +294,10 @@ class PlanarFreehandROITool extends AnnotationTool {
         label: '',
         cachedStats: {},
       },
+      ...segmentationAnnotationData,
```

Comment:
Please check if there are any before sharing. I believe it might confuse individuals who solely rely on ROI.